File: Java/Prime.java
class Prime{
    public static void main(String[] args) {
        int num = 97;
        boolean isPrime = true;

        for(int i=2;i<=num/2;i++){
            if(num%i==0){
                isPrime = false;
            }
        }
         if(isPrime == true){
            System.out.println("Prime");
         }
         else{
            System.out.println("Not Prime");
         }
    }
}

File: python/Prime.py
start=1
end=20

for i in range(start,(end+1)):
    if i>1:
        isPrime=True
        
        for j in range(2,(i//2+1)):
            if i%j==0:
                isPrime = False
                break
        
        if isPrime:
            print(i,end=" ")